terraform/main.tf

```diff
@@ -1,4 +1,7 @@
 terraform {
+  // Need a newer version of terraform in order to use count parameter in modules
+  required_version = ">= 1.0"
+
```

Comment:
```suggestion
```

We are no longer using the `count` parameter in modules, so pinning the version is no longer required.